### PR TITLE
SAWO added under stacks

### PR DESCRIPTION
**Pull request by akuadvi**
## BACKGROUND
* Add information about the background and reason for the change.

-----------------------------------------------------------------------

## Changes
* List the changes that you made

-----------------------------------------------------------------------

## Checklist
- [ ] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
- [ ] Minimum [resource requirements](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md#adding-your-application) for your application are up to date. We use this information to prevent applications from being installed on undersized clusters.
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

**Comment by paulanunda** [comment]
Hi @akuadvi, thanks for the PR. Could I get more information about this chart? Is there any documentation anywhere?

**Comment by akuadvi** [comment]
Hi Paul, you can find our doc at https://docs.sawolabs.com/sawo/ and our
official website is https://sawolabs.com/

File: stacks/SAWO/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+################################################################################
+# repo
+################################################################################
+helm repo add stable https://charts.helm.sh/stable
+helm repo update > /dev/null
+
+################################################################################
+# chart
+################################################################################
+STACK="SAWO"
```
Comment (paulanunda) [inline stacks/SAWO/deploy.sh:14]: Please change all instances of `SAWO` to `sawo`, including the directory name. Otherwise, the installation wont work.

**Review by paulanunda — CHANGES_REQUESTED** [review]
I came across the following issue while trying to install your app. Is it available as a Helm chart?

```
Release "sawo" does not exist. Installing it now.
Error: failed to download "sawo/sawo" at version "1.0.0" (hint: running `helm repo update` may help)
```

**Comment by paulanunda** [comment]
@akuadvi please answer the above question

File: stacks/SAWO/deploy.sh (hunk 1 above)
Comment (akuadvi) [reply stacks/SAWO/deploy.sh]: @paulanunda I've done the renaming in all the files but I'm unable to rename the directory.

**Comment by keladhruv** [comment]
Closing this PR due to inactivity.

**Pull request closed without merging**